### ISSUE TITLE
Errors duplicated

### ISSUE THREAD

**Issue by valeryq** [mikeerickson/validatorjs#84]
Hello. When I try execute validation.passes() several times, for example:

``` javascript
if(validation.passes()) { ... }
return validation.passes();
```

Then validation.errors duplicated as mush as many calls function (passes, fails, check).

For example:

``` javascript
validation.passes();
validation.passes();
validation.fails();
validation.errors.all(); // { login: ['The login field is required.', 'The login field is required.', 'The login field is required.']  }
```

Could you fix it?

Thank you.

**Comment by iamdtang** [comment]
Sure. Until there is a fix, save the result of the first passes() or fails() call into a variable instead of calling those methods multiple times.

**Comment by valeryq** [comment]
Already :) thanks.

**Comment by iamdtang** [comment]
Fixed in 2.0.7
